[ui5-tooling-modules] RollupError: "default" is not exported by "node_modules/raf/index.js"

**Describe the bug**
We are trying to use [jspdf](https://www.npmjs.com/package/jspdf) package with ui5-tooling-modules in ui5-ts template.
ui5-tooling-modules version is 0.7.13

When we import * as jspdf from 'jspdf' / import { jsPDF } from 'jspdf', in the console it says 'RollupError: "default" is not exported by "node_modules/raf/index.js"'

And we check the raf.js it's a CommonJS package like module.exports xxx.

**To Reproduce**
Steps to reproduce the behavior:
1. Create ts template application
2. npm i && npm i jspdf && npm run start
3. in controller import { jsPDF } from 'jspdf'
4. public onAfterRendering() {
    const pdfInstance = new jsPDF();
    // console.log(xlsx.version)
  }
5. see error in terminal

**Expected behavior**
Could use these kind of packages.

**Desktop (please complete the following information):**
 - OS: [window 10 22h2]
 - Browser [edge]

Hi @petermuessig 
Sorry for the late reply and nice to hear about having a new version release.
Now i'm working on version 0.9.3. After i remove my `node_modules` and do `npm i` and run `npm run start`. 
I'll get following error info:

Error Message:
Cannot find module 'ignore-walk'
Require stack:
- C:\Users\zoi7sgh\Documents\Project\Test\HelloWorld\ui\node_modules\ui5-tooling-modules\lib\util.js
- C:\Users\zoi7sgh\Documents\Project\Test\HelloWorld\ui\node_modules\ui5-tooling-modules\lib\middleware.js

const walk = require("ignore-walk");
I also check there is no ignore-walk package in my node_modules folder. After i `npm i -D ignore-walk`, the application started. 

Shall i create a new issue or you will do that.

Thank you very much 👍

--- a/packages/ui5-tooling-modules/package.json
+++ b/packages/ui5-tooling-modules/package.json
@@ -24,6 +24,7 @@
     "espree": "^9.5.1",
     "estraverse": "^5.3.0",
     "fast-xml-parser": "^4.2.0",
+    "ignore-walk": "^6.0.2",
     "minimatch": "^7.4.6",
     "rollup": "^3.20.2",
     "rollup-plugin-inject-process-env": "^1.3.1",

--- a/showcases/ui5-app/package.json
+++ b/showcases/ui5-app/package.json
@@ -40,11 +40,11 @@
   "dependencies": {
     "@js-temporal/polyfill": "^0.4.3",
     "@octokit/core": "4.2.0",
+    "@stomp/stompjs": "^7.0.0",
     "@supabase/supabase-js": "^2.19.0",
     "axios": "^1.3.5",
     "cmis": "^1.0.3",
     "firebase": "^9.19.1",
-    "ignore-walk": "^6.0.2",
     "node-fetch": "^2.6.9",
     "tui-image-editor": "^3.15.3",
     "ui5-lib": "^0.2.1",

--- a/showcases/ui5-app/ui5.yaml
+++ b/showcases/ui5-app/ui5.yaml
@@ -11,6 +11,9 @@ framework:
     - name: sap.ui.layout
     - name: themelib_sap_horizon
 builder:
+  componentPreload:
+    excludes:
+      - "ui5/ecosystem/demo/app/thirdparty/stomp/**"
   customTasks:
     - name: ui5-tooling-stringreplace-task
       afterTask: replaceVersion

--- a/showcases/ui5-app/webapp/controller/Thirdparty.controller.js
+++ b/showcases/ui5-app/webapp/controller/Thirdparty.controller.js
@@ -9,8 +9,9 @@ sap.ui.define(
 		"@octokit/core", // requires node-fetch@2
 		"axios",
 		"@js-temporal/polyfill",
+		"@stomp/stompjs",
 	],
-	(Controller, xlsx, cmis, _firebase, supabase, octokit, axios, temporal) => {
+	(Controller, xlsx, cmis, _firebase, supabase, octokit, axios, temporal, stompjs) => {
 		"use strict";
 
 		console.log(xlsx);
@@ -20,6 +21,7 @@ sap.ui.define(
 		console.log(octokit);
 		console.log(axios);
 		console.log(temporal);
+		console.log(stompjs);
 
 		const { initializeApp, getFirestore } = _firebase;
 		const app = initializeApp({});

--- a/showcases/ui5-tsapp/package.json
+++ b/showcases/ui5-tsapp/package.json
@@ -25,6 +25,7 @@
     "directory": "showcases/ui5-tsapp"
   },
   "dependencies": {
+    "@stomp/stompjs": "^7.0.0",
     "jspdf": "^2.5.1",
     "luxon": "^3.3.0",
     "ui5-tslib": "^0.3.6",

--- a/showcases/ui5-tsapp/ui5.yaml
+++ b/showcases/ui5-tsapp/ui5.yaml
@@ -14,6 +14,9 @@ builder:
   settings:
     includeDependency:
       - ui5.ecosystem.demo.tslib
+  componentPreload:
+    excludes:
+      - "ui5/ecosystem/demo/tsapp/thirdparty/@stomp/**"
   customTasks:
     - name: ui5-tooling-transpile-task
       afterTask: replaceVersion

--- a/showcases/ui5-tsapp/webapp/controller/Main.controller.ts
+++ b/showcases/ui5-tsapp/webapp/controller/Main.controller.ts
@@ -13,6 +13,15 @@ import("luxon")
 		console.log("Failed to load luxon from application runtime environment!", ex);
 	});
 
+// dynamic import of an existing library
+import("@stomp/stompjs")
+	.then(({ Client }) => {
+		console.log(`STOMP.js loaded: ${Client.toString()}`);
+	})
+	.catch((ex) => {
+		console.log("Failed to load STOMP.js from application runtime environment!", ex);
+	});
+
 /**
  * @namespace ui5.ecosystem.demo.tsapp.controller
  */

